docs/standard/design-guidelines/choosing-between-anonymous-and-tuple.md

```diff
@@ -104,6 +104,10 @@ While it might seem as though you should always use <xref:System.ValueTuple> ove
 | <xref:System.Tuple>      | `public`        | `class`  | ❌                   | ❌                     | ✔️                     |
 | <xref:System.ValueTuple> | `public`        | `struct` | ✔️                   | ✔️                     | ❌                     |
 
+### Serialization
+
+One important consideration when choosing a type, is whether or not it will need to be serialized. Serialization is the process of converting the state of an object into a form that can be persisted or transported. For more information, see [serialization](../../csharp/programming-guide/concepts/serialization/index.md). With anonymous types, and tuple types it is not recommended to use them if you need to serialization. While it is possible to use them with serialization, it is not fully supported, and requires workarounds.
```

Comment:
nits:

```suggestion
One important consideration when choosing a type, is whether or not it will need to be serialized. Serialization is the process of converting the state of an object into a form that can be persisted or transported. For more information, see [serialization](../../csharp/programming-guide/concepts/serialization/index.md). When serialization is important, creating a `class` or `struct` is preferred over anonymous types or tuple types.
```